x-pack/plugins/synthetics/public/apps/synthetics/components/step_details_page/components/step_metrics.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiStat,
+  EuiTitle,
+  EuiIcon,
+  EuiIconTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { CLS_HELP_LABEL, DCL_TOOLTIP, FCP_TOOLTIP, LCP_HELP_LABEL } from './labels';
+import { DefinitionsPopover } from './definitions_popover';
+import { useStepMetrics } from '../hooks/use_step_metrics';
+import { useStepPrevMetrics } from '../hooks/use_step_prev_metrics';
+
+export const formatMillisecond = (ms: number) => {
+  if (ms < 1000) {
+    return `${ms.toFixed(0)} ms`;
+  }
+  return `${(ms / 1000).toFixed(1)} s`;
+};
+
+export const StepMetrics = () => {
+  const stepMetrics = useStepMetrics();
+
+  const { fcpThreshold, lcpThreshold, clsThreshold, dclThreshold } =
+    useStepPrevMetrics(stepMetrics);
+
+  return (
+    <>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem grow>
+          <EuiTitle size="xs">
+            <h3>{METRICS_LABEL}</h3>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <DefinitionsPopover />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <EuiSpacer size="s" />
+      <EuiFlexGroup gutterSize="l">
+        <EuiFlexItem>
+          <StatThreshold
+            description={'FCP'}
+            title={formatMillisecond((stepMetrics.fcp?.value ?? 0) / 1000)}
+            threshold={fcpThreshold}
+            helpText={FCP_TOOLTIP}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <StatThreshold
+            description={'LCP'}
+            title={formatMillisecond((stepMetrics.lcp?.value ?? 0) / 1000)}
+            threshold={lcpThreshold}
+            helpText={LCP_HELP_LABEL}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <StatThreshold
+            description={'CLS'}
+            title={stepMetrics.cls?.value ?? 0}
+            threshold={clsThreshold}
+            helpText={CLS_HELP_LABEL}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiFlexGroup wrap>
+        <EuiFlexItem>
+          <StatThreshold
+            description={'DCL'}
+            title={formatMillisecond((stepMetrics.dcl?.value ?? 0) / 1000)}
+            threshold={dclThreshold}
+            helpText={DCL_TOOLTIP}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiStat
+            description={'Transfer size'}
+            title={stepMetrics.transferData + ' MB'}
+            reverse={true}
```

Comment:
We had a tooltip about transfersize as well. It's the combined `Header and Body size with compression` - https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/transferSize